src/test/addresses_deployers.sol

```diff
@@ -26,25 +26,26 @@ contract Deployers {
-    address public constant PE_04         = 0xdB33dFD3D61308C33C63209845DaD3e6bfb2c674;
-    address public constant PE_05         = 0xDA01018eA05D98aBb66cb21a85d6019a311570eE;
-    address public constant PE_06         = 0xDA0111100cb6080b43926253AB88bE719C60Be13;
-    address public constant PE_07         = 0xDa0c0De020F80d43dde58c2653aa73d28Df1fBe1;
-    address public constant PE_08         = 0xC1E6d8136441FC66612Df3584007f7CB68765e5D;
-    address public constant PE_09         = 0xa22A61c233d7242728b4255420063c92fc1AEBb9;
-    address public constant PE_10         = 0x92723e0bF280942B98bf2d1e832Bde9A3Bd2F2c2;
-    address public constant CES_01        = 0x9956fca5a8994737f124c481cEDC6BB3dc5BF010;
-    address public constant CES_02        = 0xc0b362cbb0117Ec6A4b589f744d4dECb2768A2eB;
-    address public constant CES_03        = 0xb27B6fa77D7FBf3C1BD34B0f7DA59b39D3DB0f7e;
-    address public constant CES_04        = 0x3ec4699bEc79F8FF862C220Ef0a718689A1d09f4;
-    address public constant STARKNET_01   = 0x8aa7c51A6D380F4d9E273adD4298D913416031Ec;
-    address public constant STARKNET_02   = 0x38F8e3b67FA8329FE4BaA1775e5480807f78887B;
-    address public constant CENTRIFUGE_01 = 0x9956fca5a8994737f124c481cEDC6BB3dc5BF010;
-    address public constant CENTRIFUGE_02 = 0x0A735602a357802f553113F5831FE2fbf2F0E2e0;
-    address public constant SIDESTREAM_01 = 0x47f1aaC8c1BDD49B0c2438c1754518695E9f08d3;
+    address public constant PE_01           = 0xda0fab060e6cc7b1C0AA105d29Bd50D71f036711;
+    address public constant PE_02           = 0xDA0FaB0700A4389F6E6679aBAb1692B4601ce9bf;
+    address public constant PE_03           = 0xdA0C0de01d90A5933692Edf03c7cE946C7c50445;
+    address public constant PE_04           = 0xdB33dFD3D61308C33C63209845DaD3e6bfb2c674;
+    address public constant PE_05           = 0xDA01018eA05D98aBb66cb21a85d6019a311570eE;
+    address public constant PE_06           = 0xDA0111100cb6080b43926253AB88bE719C60Be13;
+    address public constant PE_07           = 0xDa0c0De020F80d43dde58c2653aa73d28Df1fBe1;
+    address public constant PE_08           = 0xC1E6d8136441FC66612Df3584007f7CB68765e5D;
+    address public constant PE_09           = 0xa22A61c233d7242728b4255420063c92fc1AEBb9;
+    address public constant PE_10           = 0x92723e0bF280942B98bf2d1e832Bde9A3Bd2F2c2;
+    address public constant CES_01          = 0x9956fca5a8994737f124c481cEDC6BB3dc5BF010;
+    address public constant CES_02          = 0xc0b362cbb0117Ec6A4b589f744d4dECb2768A2eB;
+    address public constant CES_03          = 0xb27B6fa77D7FBf3C1BD34B0f7DA59b39D3DB0f7e;
+    address public constant CES_04          = 0x3ec4699bEc79F8FF862C220Ef0a718689A1d09f4;
+    address public constant STARKNET_01     = 0x8aa7c51A6D380F4d9E273adD4298D913416031Ec;
+    address public constant STARKNET_02     = 0x38F8e3b67FA8329FE4BaA1775e5480807f78887B;
+    address public constant CENTRIFUGE_01   = 0x9956fca5a8994737f124c481cEDC6BB3dc5BF010;
```

Comment:
`0x9956fca5a8994737f124c481cEDC6BB3dc5BF010` is both `CENTRIFUGE_01` and `CES_01`

Not sure what to do about this and the implications of renaming one (or dropping one?) of them

Reply:
hmmm... not sure either

Reply:
We will handle this in a future cleanup, but just take note of this